### PR TITLE
provide names for each format

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

File: cbor-network-addresses.mkd [hunk 1]
```diff
@@ -70,17 +70,32 @@ This specification does not deal with 6 or 8-byte Ethernet addresses.
...
 
+This form is called the Address Format.
+
+### Prefixes
+
 When applied to an array that starts with a number, they represent a
 CIDR-style prefix of that length.
-When a byte string (without prefix) appears in a context where a prefix is expected, then it is to be assumed that all bits are relevant.
+
+When an Address (byte string), i.e., without prefix) appears in a context where a prefix is expected, then it is to be assumed that all bits are relevant.
```
Comment (cabo) [inline cbor-network-addresses.mkd:86]: We can't use "address" for both the byte string and the tagged byte string.

File: cbor-network-addresses.mkd [hunk 2]
```diff
@@ -70,17 +70,32 @@ This specification does not deal with 6 or 8-byte Ethernet addresses.
...
 
+This form is called the Prefix Format.
+
+### Interface Definition
+
 When applied to an array that starts with a byte string, that stands
 for an IP address, followed by the bit length of a prefix built out of
 the first `length` bits of the address.
 
+This form is called the Interface Format, or Interface Definition.
```
Comment (cabo) [inline cbor-network-addresses.mkd:97]: Do we really need two terms (Interface format and Interface definition)?
And we don't call the other things address definition or prefix format either.

File: cbor-network-addresses.mkd (hunk 1 above)
Comment (cabo) [inline cbor-network-addresses.mkd:86]: Unbalanced parens.

File: cbor-network-addresses.mkd (hunk 2 above)
Comment (mcr) [reply cbor-network-addresses.mkd:97]: okay, so which one?

**Pull request merged**